plugins/module_utils/icinga.py

```diff
@@ -111,7 +111,7 @@ def update(self, state):
                             changed = True
                             diff_result.update({'after': 'state: absent\n'})
                         else:
-                            self.module.fail_json(msg="bad return code while deleting: %s" % (ret['data']))
+                            self.module.fail_json(msg="bad return code while creating: %s. Error message: %s" % (ret['code'], ret['msg']))
```

Comment:
HTTP resonse code should be `%d` since it is not a string
```suggestion
                            self.module.fail_json(msg="bad return code while creating: %d. Error message: %s" % (ret['code'], ret['msg']))
```

Reply:
thanks!